api_tests/lib_sdk/lnd.ts

```diff
@@ -182,31 +188,49 @@ export class Lnd {
         ).peers;
     }
 
-    public async openChannel(other: Lnd) {
-        await other.dummy();
-    }
-
-    public async addInvoice(other: Lnd) {
-        await other.dummy();
-        return "an invoice";
+    public async getChannels(): Promise<Channel[]> {
+        return (
+            await lnService.getChannels({
+                lnd: this.authenticatedLndGrpc,
+            })
+        ).channels;
     }
 
-    public async sendPayment(invoice: string) {
-        console.log("got invoice: %s", invoice);
-        await sleep(1);
+    public async openChannel(peer: Lnd, quantity: number) {
+        this.logger.debug(
+            `${this.publicKey} is opening a channel with ${peer.publicKey}; quantity: ${quantity}`
+        );
+        return lnService.openChannel({
+            lnd: this.authenticatedLndGrpc,
+            partner_public_key: peer.publicKey,
+            min_confirmations: 1,
+            local_tokens: quantity,
+        });
     }
 
-    public async assertChannelBalanceSender() {
-        await sleep(1);
+    public createInvoice(quantity: number): Promise<CreateInvoiceResponse> {
+        this.logger.debug(
+            `${this.publicKey} is creating an invoice; quantity: ${quantity}`
+        );
+        return lnService.createInvoice({
+            lnd: this.authenticatedLndGrpc,
+            tokens: quantity,
+        });
     }
 
-    public async assertChannelBalanceReceiver() {
-        await sleep(1);
+    public pay(invoice: string) {
```

Comment:
Because I did not name the return type of `ln-service.pay` but I named the return type of the other function you are looking at.
Return types are not needed in typescripts and as a team we haven't chosen a convention about it.